src/components/common/ViewPhotosModal.tsx

```diff
@@ -1,108 +0,0 @@
-import React, { useEffect, useRef, useState } from 'react';
-import strings from 'src/strings';
-import Button from 'src/components/common/button/Button';
-import DialogBox from 'src/components/common/DialogBox/DialogBox';
-import Carousel from 'react-multi-carousel';
-import 'react-multi-carousel/lib/styles.css';
-import { Box } from '@mui/material';
-
-export interface ViewPhotosDialogProps {
-  open: boolean;
-  onClose: () => void;
-  photosUrls: string[];
-  selectedSlide: number;
-}
-
-export default function ViewPhotosDialog(props: ViewPhotosDialogProps): JSX.Element {
-  const { onClose, open, photosUrls, selectedSlide } = props;
-  const [isPreviousDisabled, setIsPreviousDisabled] = useState(false);
-  const [isNextDisabled, setIsNextDisabled] = useState(false);
-
-  const myCarousel = useRef<Carousel>(null);
-  const responsive = {
-    mobile: {
-      breakpoint: { max: 4000, min: 0 },
-      items: 1,
-    },
-  };
-
-  const disableButtons = () => {
-    if (myCarousel.current) {
```

Comment:
I think it was a requirement to disable the next and previous button, is this still working?

Reply:
Oh interesting. I transferred this code directly over to the web-components version, so it should be working the same. I'll look into it. Thanks!

Reply:
It only works _after_ clicking one of those buttons. I may have messed this up—I did some things with the dialog state that might have changed this. Thanks for catching it

Reply:
I don't know if it was a requirement, I had suggested this behavior when Constanza wrote this code.